tests/lib/rules/jsx-no-target-blank.js

```diff
@@ -35,18 +35,23 @@ ruleTester.run('jsx-no-target-blank', rule, {
     {code: '<a href="foobar"></a>'},
     {code: '<a randomTag></a>'},
     {code: '<a target />'},
+    {code: '<a target={null}></a>'},
```

Comment:
this isn't a valid `target` attribute value; it might even result in a runtime warning from react.